<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlotWidget</class>
 <widget class="QWidget" name="PlotWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1310</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1310</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCustomPlot" name="plotArea" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1291</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>800</y>
     <width>1291</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="5" column="3">
     <widget class="QSlider" name="sl_ValidationB">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="15">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="8" column="0">
     <widget class="QComboBox" name="cb_tGraphDesign"/>
    </item>
    <item row="3" column="6">
     <widget class="QSpinBox" name="sb_yMin">
      <property name="maximum">
       <number>90</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="7" column="3">
     <widget class="QLineEdit" name="vGraphNameField"/>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Color of Validation Graph (r, g, b), preview:</string>
      </property>
     </widget>
    </item>
    <item row="8" column="3">
     <widget class="QComboBox" name="cb_vGraphDesign"/>
    </item>
    <item row="2" column="8">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="6">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Accuracy (%) min:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QSlider" name="sl_TrainingR">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_ColorTraining">
      <property name="text">
       <string>|||||||||||||||</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Training Graph Name:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="3">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Validation Graph Name:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QSlider" name="sl_ValidationR">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>125</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QSlider" name="sl_TrainingG">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>125</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QLabel" name="label_ColorValidation">
      <property name="text">
       <string>|||||||||||||||</string>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLineEdit" name="tGraphNameField"/>
    </item>
    <item row="2" column="5">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>50</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="4" column="3">
     <widget class="QSlider" name="sl_ValidationG">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>125</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QSlider" name="sl_TrainingB">
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="value">
       <number>125</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Color of Training Graph (r, g, b), preview:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="7">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Accuracy (%) max:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="7">
     <widget class="QSpinBox" name="sb_yMax">
      <property name="minimum">
       <number>95</number>
      </property>
      <property name="maximum">
       <number>110</number>
      </property>
      <property name="singleStep">
       <number>5</number>
      </property>
      <property name="value">
       <number>105</number>
      </property>
     </widget>
    </item>
    <item row="8" column="15">
     <widget class="QPushButton" name="btn_exportToPNG">
      <property name="text">
       <string>Export to .png</string>
      </property>
     </widget>
    </item>
    <item row="5" column="15">
     <widget class="QPushButton" name="btn_addPlotData">
      <property name="text">
       <string>Add Plot Data (.nnp)</string>
      </property>
     </widget>
    </item>
    <item row="6" column="15">
     <widget class="QPushButton" name="btn_resetPlotArea">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
